### ISSUE TITLE
Add docker image tag for Zeebe broker version.

### ISSUE THREAD

**Issue by andrejpetras** [camunda-community-hub/zeebe-hazelcast-exporter#167]
Hi,

would it be possible to add a docker image tag that only represents the Zeebe broker version?
For example, the latest Docker image has the tag `1.3.3-1.1.1-SNAPSHOT` and also `1.3.3`.

I'm working on the Quarkus Zeebe extension and I want to add this image as a devservice for the integration tests.
This additional docker tag helps me to sync Zeebe client with docker image like in `zeebe-test-container`

https://github.com/camunda-community-hub/zeebe-test-container/blob/main/src/main/java/io/zeebe/containers/ZeebeDefaults.java#L34
```
ZeebeClient.class.getPackage().getImplementationVersion()
```

Andrej

**Comment by saig0** [comment]
@andrejpetras yes. I could add an additional tag without the version of the Hazelcast exporter :+1: 

Can you tell me a bit more about your use case? Why do you want to use the Zeebe image with the Hazelcast exporter in your integration tests?

**Comment by andrejpetras** [comment]
@saig0
I use the Zeebe Image with the Hazelcast for Quarkus DevServices https://github.com/quarkiverse/quarkus-zeebe#dev-services and for integration testing https://github.com/quarkiverse/quarkus-zeebe#testing

**Comment by saig0** [comment]
@andrejpetras thank you for sharing your background :+1: 

Camunda developed a new testing framework for Zeebe: https://github.com/camunda/zeebe-process-test

Does this new official framework fulfill your requirements? Or, do you still need the Hazelcast image?

**Comment by saig0** [comment]
This is done by #189 :tada:
